### ISSUE TITLE
Anything on filesystem remote disks?

### ISSUE THREAD

**Issue by HUGE-D1CK-10-INCHES** [protonemedia/laravel-splade#177]
Custom temporary directory
By default, Splade uses the /storage/splade-temporary-file-uploads directory for temporary uploads. If you want to use a custom [Filesystem disk](https://laravel.com/docs/9.x/filesystem#configuration), you may update the file_uploads.disk key in the splade.php [configuration file](https://splade.dev/docs/customization). For now, it only supports local disks.

**Comment by HUGE-D1CK-10-INCHES** [comment]
How is this spam, when there is no answer?

**Comment by pascalbaljet** [comment]
Whoops! I thought the issue was copied over from the docs, and in combination with your username, I assumed it was spam 😅

Just to be sure, you opened this issue because you want support for temporary uploads to remote disks?

**Comment by HUGE-D1CK-10-INCHES** [comment]
Yeah, I thought of making custom uploads controllers and directing it to bunny cdn remote, since vimeo change their plans for the worse and bunny cdn is very good solution, they have storage, streaming etc. With all that, cdn has multi locations so your storage data would never be unavailable.

**Comment by HUGE-D1CK-10-INCHES** [comment]
I had some problem with uploading to bunnycdn with laravel but this repo works pretty well https://github.com/ToshY/BunnyNet-PHP
